tests/test_root_commands.py

```diff
@@ -1,4 +1,13 @@
 import pytest
+import re
+
+ATTRIBUTE_TYPES = ['int', 'bool', 'string', 'color', 'uint']
+# example values for the respective types
+ATTRIBUTE_VALUES = [[23, 42, -8],
+                    ['true', 'false'],
+                    ['foo', 'baz', 'bar'],
+                    ['#ff00ff', '#9fbc00'],  # FIXME: include named colors
+                    [23, 42]]
```

Comment:
Why not structure these as a dictionary? Having two independent lists with some implicit relation between their orders is just unnecessarily hard to read and maintain, in my opinion.

I suspect you did it like this so that you can use the globals for parametrization (as-is), but it don't immediately see why it wouldn't work with a dictionary's `.items()` list as well.

Reply:
Right, a dictionary is better